pkgs/development/lisp-modules/quicklisp-to-nix-output/cl-custom-hash-table.nix

```diff
@@ -0,0 +1,25 @@
+args @ { fetchurl, ... }:
+rec {
+  baseName = ''cl-custom-hash-table'';
+  version = ''20201220-git'';
```

Comment:
Please use " for single line strings.

Reply:
Hm, the change creates only one case of escape use in the quicklisp-to-nix output, so can as well regenerate.